Web/src/components/games/PandaSequence/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
+import { share } from 'rxjs/operators';
+import { Sequence, PandaSequenceMode } from './Sequence';
+import { createTimerObservable } from '../rxhelpers';
+import { GameState } from '../BalloonShake/GameStates';
+import GameResults from './GameResults';
+import GameDisplay from './GameDisplay';
+import useSeqGenerator from './useSeqGenerator';
+
+const GAME_TIME = 25;
+const SEED = '100';
+const NUM_POTS = 5;
+
+const PandaSequence: React.FC = () => {
+  const [gameState, setGameState] = useState(GameState.WAITING_START);
+  const [mode, setMode] = useState(PandaSequenceMode.DISPLAY);
+  const [secondsLeft, setSecondsLeft] = useState(GAME_TIME);
+  const [score, setScore] = useState(0);
+  const [sequence, setSequence] = useState<Sequence>({ timestep: 0, numbers: [] });
+  const [index, setIndex] = useState(0);
+  const [inputIndex, setInputIndex] = useState(0);
+  const { generate } = useSeqGenerator(SEED, 0, NUM_POTS);
+
+  /** Setup game and trigger first sequence */
+  useEffect(() => {
+    const timer = createTimerObservable(GAME_TIME).pipe(share());
+    timer.subscribe(
+      left => setSecondsLeft(left),
+      null,
+      () => setGameState(GameState.WAITING_RESULTS),
+    );
+    setGameState(GameState.IN_PROGRESS);
+  }, []);
+
+  /** Display new sequence */
+  useEffect(() => {
+    if (sequence.numbers.length === 0) return;
+
+    const { timestep, numbers } = sequence;
+    const timer = createTimerObservable(numbers.length + 1, timestep).pipe(share());
```

Comment:
actually don't need to share if you're not sharing it.
```suggestion
    const timer = createTimerObservable(numbers.length + 1, timestep);
```

Reply:
Sure thing. I read the docs for [multicasting](https://www.learnrxjs.io/operators/multicasting/share.html) and couldn't see why the `share` was required in BalloonShake, so I included it conservatively. Works fine without it so I'll be removing!